### PR TITLE
[Improvement] Fixed RTL layout for change Password

### LINKED ISSUE #1045: [BUG] Change password layout showing empty view in RTL layout
Your Rocket.Chat.Android version:
- versionCode 2007
- versionName "2.0.0-beta5"

Mobile device model and OS version: Nexus 6 - Android 7.1
device-2018-04-07-182735

### PR DESCRIPTION
**Pull request by divyanshub024**
@RocketChat/android

Closes #1045 

pass-rtl-layout

### CODE REVIEW

File: app/src/main/res/layout/fragment_password.xml [hunk 1]
```diff
@@ -2,32 +2,38 @@
...
         android:layout_height="wrap_content"
         android:theme="@style/EditText.Password"
-        app:layout_constraintBottom_toTopOf="@id/middle_guide"
-        app:layout_constraintStart_toEndOf="@id/start_guide"
-        app:layout_constraintEnd_toStartOf="@id/end_guide">
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:layout_margin="16dp"
+        >
```
Comment (philipbrito) [inline app/src/main/res/layout/fragment_password.xml:17]: This `>` could be on the same line of `android:layout_margin="16dp"`

File: app/src/main/res/layout/fragment_password.xml [hunk 2]
```diff
@@ -2,32 +2,38 @@
...
         android:layout_height="wrap_content"
         android:theme="@style/EditText.Password"
-        app:layout_constraintTop_toBottomOf="@id/middle_guide"
-        app:layout_constraintStart_toEndOf="@id/start_guide"
-        app:layout_constraintEnd_toStartOf="@id/end_guide">
+        app:layout_constraintTop_toBottomOf="@+id/layout_new_password"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_margin="16dp"
+        >
```
Comment (philipbrito) [inline app/src/main/res/layout/fragment_password.xml:36]: This `>` could be on the same line of `android:layout_margin="16dp"`

**Review by philipbrito — APPROVED** [review]
Just some code formatting that could be addressed here but will go in another PR.
Thanks for this PR.

File: app/src/main/res/layout/fragment_password.xml (hunk 2 above)
Comment (divyanshub024) [reply app/src/main/res/layout/fragment_password.xml:36]: @filipedelimabrito I will keep this in mind next time.

**Pull request merged**